iceoryx_posh/test/integrationtests/test_service_discovery.cpp

```diff
@@ -100,348 +123,399 @@ void compareServiceContainers(const ServiceContainer& lhs, const ServiceContaine
     }
 }
 
-TEST_F(ServiceDiscovery_test, FindServiceWithWildcardsReturnsOnlyIntrospectionServicesAndServiceRegistry)
-{
-    ::testing::Test::RecordProperty("TEST_ID", "d944f32c-edef-44f5-a6eb-c19ee73c98eb");
-    auto searchResult = sut.findService(iox::capro::Wildcard, iox::capro::Wildcard, iox::capro::Wildcard);
-
-
-    for (auto& service : searchResult)
-    {
-        EXPECT_THAT(service.getInstanceIDString().c_str(), StrEq("RouDi_ID"));
-    }
+///
+/// Publish-Subscribe & Request-Response
+///
 
-    sut.findService(iox::capro::Wildcard, iox::capro::Wildcard, iox::capro::Wildcard, findHandler);
-    compareServiceContainers(searchResult, searchResultOfFindServiceWithFindHandler);
-}
+using CommunicationKind = Types<Publisher, Server>;
+TYPED_TEST_SUITE(ServiceDiscovery_test, CommunicationKind);
```

Comment:
@MatthiasKillat @FerdinandSpitzschnueffler @elBoberido 

We'll get merge conflicts in this file. I don't mind solving these and would propose the following merge order: 

1. #1105 
2. #1127 
3. #1091 

After that I'll tackle #1065.

What do you think?

Reply:
Sounds reasonable